include/shell/shell_record.h

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
```

Comment:
I am not entirely sure that Nordic has participated in this file ;-)

Reply:
I copied the file from shell_dummy so thought I should credit it?

Reply:
I've written shell_dummy and I do not mind :)